tet/security/authentication.py

```diff
@@ -434,26 +522,65 @@ def __init__(self, request: Request):
         self.long_term_token_header = self.registry.tet_auth_long_term_token_header
         self.access_token_header = self.registry.tet_auth_access_token_header
         self.project_prefix = self.registry.tet_auth_project_prefix
+        self.access_token_cookie_name = self.registry.tet_auth_access_token_cookie_name
+        self.long_term_token_cookie_name = self.registry.tet_auth_long_term_token_cookie_name
 
-    def login_view(self) -> tp.Union[tp.Dict[str, tp.Any], HTTPForbidden]:
-        login_callback = self.registry.tet_auth_login_callback
+    def _set_cookie(
+        self,
+        name,
+        value,
+        max_age,
+        domain=None,
+        secure=True,
+        httponly=True,
+        samesite="Lax",
+        overwrite=True,
+        **kwargs,
+    ):
+        self.response.set_cookie(
+            name=name,
+            value=value,
+            max_age=max_age,
+            domain=domain,
+            secure=secure,
+            httponly=httponly,
+            samesite=samesite,
+            overwrite=overwrite,
+            **kwargs,
+        )
 
+    def login(self) -> tp.Union[tp.Dict[str, tp.Any], HTTPForbidden, None]:
+        login_callback = self.registry.tet_auth_login_callback
         user_id = login_callback(self.request)
 
         if user_id is None:
-            raise HTTPForbidden()
+            raise HTTPUnauthorized()
```

Comment:
Indeed, unauthorized. Maybe should include a message, provide body in JSON format thouhg.